src/utils/cameras.utils.ts

```diff
@@ -0,0 +1,18 @@
+export const fadeCameraToScene = (
```

Comment:
Instead of switching scenes by `this.scene.start(...)` for a more aesthetic scene switch `fadeCameraToScene(...)` should be used.